helm/charts/determined/templates/db-persitent-volume-claim.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: determined-db-pvc-{{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: determined-db-{{ .Release.Name }}
+    release: {{ .Release.Name }}
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 30Gi
```

Comment:
non-blocking: if we decide to have the postgres helm install as part of our chart we might want to make this field configurable since DB's can get quite large.

Reply:
Yep good call, made this configurable.